src/Orleans.Http.Abstractions/IRouteGrainProvider.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using System.Threading.Tasks;
+
+namespace Orleans.Http.Abstractions
+{
+    public interface IRouteGrainProvider
+    {
+        Task<IGrain> GetGrain(Type grainType);
```

Comment:
Since this will not _always_ be an async operation, can you please change it to `ValueTask<IGrain>`?

Reply:
Returning type is now a ValueTask<IGrain>